sopel/test_tools.py

```diff
@@ -22,6 +22,7 @@
 import sopel.config
 import sopel.config.core_section
 import sopel.tools
+import sopel.tools.target
```

Comment:
This is done in the `6.6.x` branch already (from #1529), and will cause a merge conflict.

Reply:
Not the worst offender. :D